### ISSUE TITLE
Unable to access FundamentalData from Python Notebook

### ISSUE THREAD

**Issue by Parkman328** [RomelTorres/alpha_vantage#271]
Installed Alpha Vantage from Pip and Fundamental Data is unable to be updated.

'from alpha_vantage import FundamentalData'

ModuleNotFoundError: No module named 'alpha_vantage.fundamentaldata'

**Comment by scdanieli** [comment]
There is no `FundamentalData` class  in the latest release (2.2.0).

Workaround:
```bash
# https://coderwall.com/p/-wbo5q/pip-install-a-specific-github-repo-tag-or-branch
pip3 install -e git://github.com/RomelTorres/alpha_vantage.git@develop#egg=alpha_vantage
```
Besides that, it's ...
```python
from alpha_vantage.fundamentaldata import FundamentalData
```
not 
```python
from alpha_vantage import FundamentalData
```

**Comment by PatrickAlphaC** [comment]
@scdanieli how has working with the fundamental data been? Pretty smooth/seamless?

**Comment by PatrickAlphaC** [comment]
This has been addressed in 2.3.1. Looking for help adding docs!

**Comment by scdanieli** [comment]
@PatrickAlphaC Haven't had any issues with it, but I've only worked with `get_company_overview` so far.
